gcloud/bigquery/dataset.py

```diff
@@ -64,6 +64,8 @@ def __init__(self, name, client, access_grants=()):
         self.name = name
         self._client = client
         self._properties = {}
+        self._access_grants = None
+        # Let the @property do validation.
```

Comment:
This dead chicken is dubious:  the property setter immediately following will overwrite it.